### PR TITLE
curl import added

### PR DESCRIPTION
**Pull request by nityanandagohain**
Fixes issue #93 

Hey @liyasthomas  I have tried to add functionality to import curl commands. Please let me know if any changes are required.

Screenshot from 2019-09-06 16-17-42
Screenshot from 2019-09-06 16-19-49
Screenshot from 2019-09-06 16-23-00

### CODE REVIEW

File: pages/index.vue [hunk 1]
```diff
@@ -1,6 +1,19 @@
 <template>
   <div class="page">
-    <pw-section class="cyan" label="Request" ref="request">
+    <pw-section class="blue" label="Request" ref="request">
+      <button id="show-modal" @click="showModal = true">IMPORT</button>
+      <import-modal v-if="showModal" @close="showModal = false">
+        <div slot="header">
+          <h2>IMPORT CURL</h2>
```
Comment (NBTX) [inline pages/index.vue]: Can you please make this title case?
'Import cURL'

File: pages/index.vue [hunk 2]
```diff
@@ -1,6 +1,19 @@
 <template>
   <div class="page">
-    <pw-section class="cyan" label="Request" ref="request">
+    <pw-section class="blue" label="Request" ref="request">
+      <button id="show-modal" @click="showModal = true">IMPORT</button>
```
Comment (NBTX) [inline pages/index.vue]: Can you rename this to 'Import' and move it outside of the `pw-section` for request?

Comment (nityanandagohain) [reply pages/index.vue]: @NBTX  I have made the requested changes, please have a look.

File: pages/index.vue [hunk 3]
```diff
@@ -1,10 +1,10 @@
 <template>
   <div class="page">
-    <pw-section class="blue" label="Request" ref="request">
+    <pw-section class="blue" label="Import" ref="import">
       <button id="show-modal" @click="showModal = true">IMPORT</button>
```
Comment (NBTX) [inline pages/index.vue]: Can you change this from 'IMPORT' to 'Import'?
Other than that, everything's perfect.

Comment (nityanandagohain) [reply pages/index.vue]: Done.

**Comment by liyasthomas** [comment]
@NBTX merge after your approval

**Comment by liyasthomas** [comment]
@nityanandagohain @hosseinnedaee 
Ran into a bug in cURL parser! After importing a cURL, `PATH` is appended to `URL` which is okay. But tje existing `PATH` field stays same. We manually need to empty `PATH` field before sending next request.
To reproduce:
1. Generate a cURL code (⚡ icon under URL field) for the default test request
2. Copy thc cURL and import the same using Import cURL button
3. See URL is appended with PATH as well as the existing PATH stays as same as before
Fix:
Either replace the existing PATH field with the one from the imported cURL (which is the best UX)
Or
Append PATH to URL, but empty the existing PATH field.

I wonder why anybody haven't reported this yet. And need to look at error handler on index.vue regarding importCURL as well since from 2nd import cURL request, function lands me on error handler shooting error toast. 
https://github.com/liyasthomas/postwoman/blob/master/assets/js/curlparser.js

**Pull request merged**